pallets/loans/src/lib.rs

```diff
@@ -703,7 +709,7 @@ pub mod pallet {
 				for loan in active_loans.iter() {
 					weight += T::DbWeight::get().reads_writes(1, 1);
 					let rate = Self::rate_with_penalty(&loan, &write_off_groups);
-					T::InterestAccrual::reference_rate(rate);
+					T::InterestAccrual::reference_rate(rate).unwrap();
```

Comment:
This migration is not versioned. So basically, this will run everytime we run a runtime upgrade.  
We should version. 

Not sure, but it probably ran once already, right? 

Also, I am on the fence of having migration code in the normal pallet.

Reply:
This migration was a big of a mess due to needing to coordinate data between interest-accrual and the loans pallet. It's already been applied at this point, and just needs to be deleted in a future PR.